src/main/java/org/dita/dost/writer/ForceUniqueFilter.java

```diff
@@ -55,20 +69,35 @@ public void startElement(final String uri, final String localName, final String
             final String format = res.getValue(ATTRIBUTE_NAME_FORMAT);
             // FIXME: handle cascading
             final String processingRole = res.getValue(ATTRIBUTE_NAME_PROCESSING_ROLE);
+            FileInfo dstFi = null;
             if (source != null &&
                     isLocalScope(scope) &&
                     (format == null || format.equals(ATTR_FORMAT_VALUE_DITA)) &&
                     (processingRole == null || processingRole.equals(ATTR_PROCESSING_ROLE_VALUE_NORMAL))) {
                 final URI file = stripFragment(source);
                 Integer count = topicrefCount.getOrDefault(file, 0);
-                count++;
-                topicrefCount.put(file, count);
+                
+                ParentTopicref parentTopicref = topicrefParentsStack.peek();
+                boolean parentTopicHasEmbededSubtopcis = false;
+                if(parentTopicref != null && parentTopicref.getHref() != null) {
```

Comment:
Add space between `if` and `(`, here and in other places

Reply:
I made the requested changes